### DISCUSSION TITLE
why you ignore extension name (ex.name) inside mods folder?

### DISCUSSION THREAD

**Discussion by fr1g** [FabricMC/fabric-api#3686] [Player support] [answered]
### Minecraft Logs

```txt
there's no need to paste logs.
```

### Description

Our launcher can switch a mod into "disabled" state to prevent it from loading stage by changing its ex.name from "jar" to "disabled", which theoretically can make debugging modpack way more convenient. but seems fabric will ignore every single file's ex.name and try to read and load them as mods. this is quite weird! why!
and if the duplicated mods existing with same file names but only ex.name different, fabric will try to load twice?!
why you do so?

**Comment by Linguardium** [comment, marked as answer]
fabric does not ignore extensions. this disabling method is used by prism launcher and it works perfectly fine with fabric mod loader

**Comment by fr1g** [comment]
That’s weird! Why the same method not working on my launcherAny arguments can make influence?It’s not likely that they have something has a build cache🤯Отправлено с iPhone4 апр. 2024 г., в 19:00, Linguardium ***@***.***> написал(а):﻿
fabric does not ignore extensions. this disabling method is used by prism launcher and it works perfectly fine with fabric mod loader
